neuromation/cli/admin.py

```diff
@@ -209,6 +210,25 @@ def admin() -> None:
         )
 
 
+def _parse_quota_value(
+    value: Optional[str], allow_infinity: bool = False
+) -> Optional[int]:
+    if value is None:
+        return None
+    try:
+        result = float(value[:-1]) * {"h": 60, "m": 1}[value[-1]]
```

Comment:
Please improve the error reporting; `_parse_quota_value("5z")` generates obscure `KeyError: 'z'` exception.